plugins/fcp_cpp/tests/schemas/test.fcp

```diff
@@ -18,70 +12,78 @@ enum E {
 }
 
 struct S2 {
-    s1 @ 0: u8,
```

Comment:
grrr, exactly the same issue as with the hex numbers 😳.

I've created this issue https://github.com/joajfreitas/fcp-core/issues/255